### ISSUE TITLE
Json is no longer valid in latest version of Intune

### ISSUE THREAD

**Issue by ojrivera381** [damienvanrobaeys/Intune-Reporting#2]
The BIOSUpdate_Workbook.json is not valid see attached screenshot.
![JsonError](https://github.com/damienvanrobaeys/Intune-Reporting/assets/16689299/db112c5b-6dde-44f1-8562-2248041165b9)

**Comment by Mon3yb** [comment]
I just imported the Workbook yesterday. Everything worked as expected. If I take the error message in the screenshot on face value it would suggest that you tried to import it into a single field in the workbook.
![image](https://github.com/damienvanrobaeys/Intune-Reporting/assets/2087287/15288da4-98df-44f0-972c-1b6adda9de7b)

**Comment by ojrivera381** [comment]
Yes are correct, when I clicked on that icon I pasted the whole json in. Am I supposed to add each type individually?

**Comment by ojrivera381** [comment]
Thank you for the assist here. Much appreciated. Ok, So I added each type individually. Progress, but now I'm getting the following errors for each of them. Line numbers are different see screenshot. 
![Query Errors](https://github.com/damienvanrobaeys/Intune-Reporting/assets/16689299/c7a78bd1-0b94-42cd-9054-c25964a3630d)

> Query could not be parsed at ')' on line [6,36]
Token: ')' Line: 6 Postion: 36

**Comment by Mon3yb** [comment]
I think I was a bit unclear. Importing it through that button on the top of a workbook would be the correct way. If that doesn't work then I can't really help any further :(
